src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1192,7 +1193,10 @@ private async Task<TokenValidationResult> ValidateTokenAsync(string token, JsonW
 
```

Comment:
Since you are at it, can you change this to use await instead of GetAwaiter (same as the call above)? The method can throw an exception so a try/catch is needed here.Because this is used multiple times, consider wrap it in a helper method so it is consistent.